repositories.bzl

```diff
@@ -351,3 +351,12 @@ py_proto_library(
 def mixerapi_dependencies():
     go_x_tools_imports_repositories()
     mixerapi_repositories()
+
+def docker_dependencies():
+    # Download the rules_docker repository at release v0.12.0
```

Comment:
Nit: This comment is a little redundant with the next few lines.

Reply:
Removed it.